### DISCUSSION TITLE
Custom exporters

### DISCUSSION THREAD

**Discussion by mrpantsuit** [open-telemetry/opentelemetry-collector#2894] [Q&A]
We'd like to export our metrics to Prometheus, but also export a sub-sampling of those metrics to Google BigQuery. Would it be possible to write a custom exporter to do that? Thx.

**Comment by bogdandrutu** [comment]
You can do that right now, by setting up 2 pipelines from the receivers and using https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/filterprocessor processor to filter the metrics you want. On the Prometheus pipeline you don't filter anything but on the google pipeline filter that sub-set you need
